### PR TITLE
Lt 1177 users pagination

### PR DESCRIPTION
**Pull request by a-bek**

### CODE REVIEW

File: core/services.py [hunk 1]
```diff
@@ -122,8 +123,8 @@ def get_organisation(request, pk):
     return data.json()
 
 
-def get_organisation_users(request, pk):
-    data = get(request, ORGANISATIONS_URL + pk + "/users/")
+def get_organisation_users(request, pk, params):
+    data = get(request, ORGANISATIONS_URL + pk + "/users/?" + urlencode(params))
```
Comment (BenAndrew-Kainos) [inline core/services.py]: nice

File: users/views.py [hunk 2]
```diff
@@ -29,12 +34,23 @@ def get(self, request, **kwargs):
         if organisation["type"]["key"] == "individual":
             raise Http404
 
+        # Return from API and swap key and value names.
+        statuses = [
+            Option(option["key"], option["value"])
+            for option in [{"key": "active", "value": "Active"}, {"key": "", "value": "All"}]
```
Comment (BenAndrew-Kainos) [inline users/views.py]: Same comments as in internal (https://github.com/uktrade/lite-internal-frontend/pull/362#discussion_r379521191)

File: pipeline/Jenkinsfile [hunk 3]
```diff
@@ -138,7 +138,7 @@ def sendSlackNotification()
         emoji = "🔥"
     }
 
-	def message = "${emoji}   UAT ${buildStatus} \n📇   *lite-internal-frontend* \n🔗   ${env.BUILD_URL}"
+	def message = "${emoji}   UAT ${buildStatus} \n📇   *lite-exporter-frontend* \n🔗   ${env.BUILD_URL}"
```
Comment (phlptt) [inline pipeline/Jenkinsfile]: 👍🧘‍♂️

Comment (jfaracik) [reply pipeline/Jenkinsfile]: Fix the emoji please!

**Pull request merged**